contracts/utils/cryptography/SignatureChecker.sol

```diff
@@ -121,6 +121,28 @@ library SignatureChecker {
         }
     }
 
+    /**
+     * @dev Variant of {isValidSignatureNow-bytes-bytes32-bytes-} that takes a signature in calldata
+     */
+    function isValidSignatureNowCalldata(
+        bytes calldata signer,
+        bytes32 hash,
+        bytes calldata signature
+    ) internal view returns (bool) {
+        if (signer.length < 20) {
+            return false;
+        } else if (signer.length == 20) {
+            return isValidSignatureNowCalldata(address(bytes20(signer)), hash, signature);
+        } else {
+            (bool success, bytes memory result) = address(bytes20(signer)).staticcall(
+                abi.encodeCall(IERC7913SignatureVerifier.verify, (signer[20:], hash, signature))
```

Comment:
I'm thinking it may not makes sense to receive the signer and signature from calldata if they'll be encoded for the verifier call.